daemon/images/image_builder.go

```diff
@@ -170,6 +170,9 @@ func (i *ImageService) pullForBuilder(ctx context.Context, name string, authConf
 func (i *ImageService) GetImageAndReleasableLayer(ctx context.Context, refOrID string, opts backend.GetImageAndLayerOptions) (builder.Image, builder.ROLayer, error) {
 	if refOrID == "" { // ie FROM scratch
 		os := runtime.GOOS
+		if runtime.GOOS == "windows" {
```

Comment:
Just checking here to make sure I understand, if we have an empty `refOrID` at this point we are  assuming it's LCOW?

Reply:
Yep. Windows containers does not support scratch.

Reply:
> Just checking here to make sure I understand, if we have an empty refOrID at this point we are  assuming it's LCOW?

Correct on the Windows path.